metatron-app/metatron-discovery
# #3623 Engine health check notification

### Description

When the engine state has changed, Send a notification to Slack.
 
**Related Issue** : #3623



### How Has This Been Tested?

1. Add properties related to sending slack to application.yaml.
```
polaris:
  engine:
    notification:
      type: historical
      slack:
        hookUrl:
        channel:
```
2. Change the engine state and check if slack notification is received.
![Slack___notification-test___Polaris](https://user-images.githubusercontent.com/45194478/111121435-fb7d5c80-85af-11eb-9d77-fc6304e7083b.jpg)

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
